src/Console/Command/ListMetricsCommand.php

```diff
@@ -72,5 +72,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $table->render();
+
+        return Command::SUCCESS;
```

Comment:
Could you please use `\Magento\Framework\Console\Cli::RETURN_SUCCESS` here?

Reply:
Done!